illuminate/cookie/2020-07-27-1.yaml

```diff
@@ -2,9 +2,39 @@ title:     RCE vulnerability in "cookie" session driver
 link:      https://blog.laravel.com/laravel-cookie-security-releases
 cve:       ~
 branches:
+    "4.1":
+        time:     ~
+        versions: ['>=4.1.0', '<=4.1.99999']
+    "5.2":
+        time:     ~
+        versions: ['>=4.2.0', '<=4.2.99999']
+    "5.0":
+        time:     ~
+        versions: ['>=5.0.0', '<=5.0.99999']
+    "5.1":
+        time:     ~
+        versions: ['>=5.1.0', '<=5.1.99999']
+    "5.2":
```

Comment:
5.2 is also duplicated in this file.

Reply:
Not duplicated, but still wrong. The first occurrence was a typo for 4.2. I'll get that fixed. ;)